TeamSnapSDK/SDK/Cache/TSPCache.m

```diff
@@ -22,16 +22,26 @@ +(void)setCacheRootPath:(NSURL *)rootPath {
 
 +(NSURL *)cacheRootPath {
     if (!_rootPath) {
-        NSString *directory = [NSSearchPathForDirectoriesInDomains(NSDocumentDirectory, NSUserDomainMask, YES) lastObject];
+        NSString *directory = [NSSearchPathForDirectoriesInDomains(NSCachesDirectory, NSUserDomainMask, YES) lastObject];
         _rootPath = [[NSURL fileURLWithPath:directory isDirectory:YES] URLByAppendingPathComponent:@"teamsnap"];
     }
     if (![[self fileManager] fileExistsAtPath:[_rootPath path]]) {
         [[self fileManager] createDirectoryAtPath:[_rootPath path] withIntermediateDirectories:YES attributes:nil error:nil];
+        [self markDirectoryForNoBackup:_rootPath];
```

Comment:
@larsacus actually, with the .app branch, the app can function without a network connection (for viewing data, only. No Edits/Adds/Deletes). 🔥🤘